meilisearch/client.py

```diff
@@ -265,6 +254,85 @@ def get_keys(self) -> Dict[str, str]:
         """
         return self.http.get(self.config.paths.keys)
 
+    def create_key(
+        self,
+        options: Dict[str, Any]
+    ) -> Dict[str, Any]:
+        """Creates a new API key.
+
+        Parameters
+        ----------
+        options:
+            Options, the information to use in creating the key (ex: { 'actions': ['*'], 'indexes': ['movies'], 'description': 'Search Key', 'expiresAt': '22-01-01' }).
+            An `actions`, an `indexes` and a `expiresAt` fields are mandatory,`None` should be specified for no expiration date.
+            `actions`: A list of actions permitted for the key. ["*"] for all actions.
+            `indexes`: A list of indexes permitted for the key. ["*"] for all indexes.
+            Note that if an expires_at value is included it should be in UTC time.
+
+        Returns
+        -------
+        keys:
+            The new API key.
+            https://docs.meilisearch.com/reference/api/keys.html#get-keys
+
+        Raises
+        ------
+        MeiliSearchApiError
+            An error containing details about why MeiliSearch can't process your request. MeiliSearch error codes are described here: https://docs.meilisearch.com/errors/#meilisearch-errors
+        """
+        return self.http.post(f'{self.config.paths.keys}', options)
+
+    def udpate_key(
```

Comment:
I just noticed that there is a typo here. Should be `update_key` instead of `udpage_key`.

Reply:
Oh! Good catch thank you @sanders41 😄